ingest/src/main/java/com/microsoft/azure/kusto/ingest/IngestionProperties.java

```diff
@@ -292,30 +292,32 @@ void validate() {
     }
 
     public enum DataFormat {
-        csv(IngestionMapping.IngestionMappingKind.Csv, false),
-        tsv(IngestionMapping.IngestionMappingKind.Csv, false),
-        scsv(IngestionMapping.IngestionMappingKind.Csv, false),
-        sohsv(IngestionMapping.IngestionMappingKind.Csv, false),
-        psv(IngestionMapping.IngestionMappingKind.Csv, false),
-        txt(IngestionMapping.IngestionMappingKind.unknown, false),
-        tsve(IngestionMapping.IngestionMappingKind.Csv, false),
-        json(IngestionMapping.IngestionMappingKind.Json, true),
-        singlejson(IngestionMapping.IngestionMappingKind.Json, true),
-        multijson(IngestionMapping.IngestionMappingKind.Json, true),
-        avro(IngestionMapping.IngestionMappingKind.Avro, true),
-        apacheavro(IngestionMapping.IngestionMappingKind.ApacheAvro, false),
-        parquet(IngestionMapping.IngestionMappingKind.Parquet, false),
-        sstream(IngestionMapping.IngestionMappingKind.SStream, false),
-        orc(IngestionMapping.IngestionMappingKind.Orc, false),
-        raw(IngestionMapping.IngestionMappingKind.unknown, false),
-        w3clogfile(IngestionMapping.IngestionMappingKind.W3CLogFile, false);
+        csv(IngestionMapping.IngestionMappingKind.Csv, false, true),
+        tsv(IngestionMapping.IngestionMappingKind.Csv, false, true),
+        scsv(IngestionMapping.IngestionMappingKind.Csv, false, true),
+        sohsv(IngestionMapping.IngestionMappingKind.Csv, false, true),
+        psv(IngestionMapping.IngestionMappingKind.Csv, false, true),
+        txt(IngestionMapping.IngestionMappingKind.unknown, false, true),
+        tsve(IngestionMapping.IngestionMappingKind.Csv, false, true),
+        json(IngestionMapping.IngestionMappingKind.Json, true, true),
+        singlejson(IngestionMapping.IngestionMappingKind.Json, true, true),
+        multijson(IngestionMapping.IngestionMappingKind.Json, true, true),
```

Comment:
This had been treated as **not** being a mappingRequiredFormat. But I believe that's a bug.